packages/dds/merge-tree/src/test/client.rollbackFarm.spec.ts

```diff
@@ -19,12 +19,12 @@ import { createClientsAtInitialState, TestClientLogger } from "./testClientLogge
 
 const allOperations: TestOperation[] = [
     removeRange,
-    // annotateRange, // Bug AB2724: properties in rollback client don't match oracle
+    annotateRange, // Bug AB2724: properties in rollback client don't match oracle
     insertAtRefPos,
 ];
 
 const defaultOptions = {
-    minLength: { min: 1, max: 32 },
+    minLength: { min: 1, max: 16 /* 32 */ },
```

Comment:
what's this change for?

Reply:
Enabling annotateRange with minLength = 32 caused the insert failed bug (mentioned on the next line) to happen with minLength=32. I think it's just the randomness.